tools/tox.ini

```diff
@@ -42,26 +42,26 @@ commands = flake8 --append-config={toxinidir}/py3-flake8.ini {posargs}
 deps = -rrequirements_flake8.txt
 commands = flake8 --append-config={toxinidir}/py3-flake8.ini {posargs}
 
-[testenv:py35-mypy]
+[testenv:py35-mypy2]
 deps = -rrequirements_mypy.txt
 changedir = {toxinidir}/..
 commands =
   mypy --config-file={toxinidir}/mypy.ini --no-incremental --py2 -p tools.manifest -p tools.lint -p tools.gitignore
 
-[testenv:py36-mypy]
+[testenv:py38-mypy2]
 deps = -rrequirements_mypy.txt
 changedir = {toxinidir}/..
 commands =
   mypy --config-file={toxinidir}/mypy.ini --no-incremental --py2 -p tools.manifest -p tools.lint -p tools.gitignore
 
-[testenv:py37-mypy]
+[testenv:py35-mypy3]
 deps = -rrequirements_mypy.txt
 changedir = {toxinidir}/..
 commands =
-  mypy --config-file={toxinidir}/mypy.ini --no-incremental --py2 -p tools.manifest -p tools.lint -p tools.gitignore
+  mypy --config-file={toxinidir}/mypy.ini --no-incremental -p tools.manifest -p tools.lint -p tools.gitignore
 
-[testenv:py38-mypy]
+[testenv:py38-mypy3]
 deps = -rrequirements_mypy.txt
 changedir = {toxinidir}/..
 commands =
-  mypy --config-file={toxinidir}/mypy.ini --no-incremental --py2 -p tools.manifest -p tools.lint -p tools.gitignore
+  mypy --config-file={toxinidir}/mypy.ini --no-incremental -p tools.manifest -p tools.lint -p tools.gitignore
```

Comment:
Could you elaborate on how this combination was chosen?

I don't think there's a difference between `py35-mypy2` and `py38-mypy2`, both set `--python-version=2.7` under the hood and hence use the same typing information. If there's any inconsistency, we'll actually be discovering bugs in mypy itself.

Also, it'd be great to keep the 3.6 and 3.7 coverage. In other words, we should have `mypy --python-version` covering 2.7, 3.5, 3.6, 3.7 & 3.8.

Reply:
> Well we didn't have any Python 3 coverage before, so "keep".

But tox.ini currently has `{py35,py36,py37,py38}-mypy`?

> But I thought it made sense to run with the lowest and highest supported versions and assume that would be enough to cover the range.

This is a reasonable expectation, but personally I'd err on the cautious side. Anyway, it is not a big deal in practice.

Reply:
Ahh sorry you're right.